coalesce/config/common.py

```diff
@@ -15,6 +15,7 @@ class Common(Configuration):
         'django.contrib.sessions',
         'django.contrib.messages',
         'django.contrib.staticfiles',
+        'django.contrib.postgres',
```

Comment:
Is this necessary for `search_fields = ['@FIELD']` to work?

Reply:
Yep it seems so, when I remove it and re-run the tests I get this exception:
```python
ERROR Internal Server Error: /api/v1/opportunities/itiesListTestCase.test_post_request_with_query_params_returns_subset            ____________
Traceback (most recent call last):                                                                                                             
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner    
  .......
  django.core.exceptions.FieldError: Unsupported lookup 'search' for CharField or join on the field not permitted.
```

Reply:
Got it, thanks for checking!